config.env

```diff
@@ -1,2 +1,3 @@
-DB_URL = postgres://acmt:12345678@localhost:5432/acmt
-SECRET = AbuMohammed
+DB_URL = postgres://four:123@localhost:5432/four
+TEST_DB_URL = postgres://fourtest:123@localhost:5432/fourtest
```

Comment:
Are you sure you added `config.env` to `.gitignore`?

Reply:
Yes, I'm very sure I don't know what's wrong.

Reply:
I'll remove the environmental variables from the web editor, please don't let them come through again.